contracts/phoenix-bonds/src/accrual.rs

```diff
@@ -108,9 +108,16 @@ impl AccrualParameter {
         self.mean_length.insert(amount, ts); // this could only make mean length shorter
         let new_mean_length = self.mean_length.mean(ts);
 
-        if old_mean_length >= self.target_mean_length && new_mean_length < self.target_mean_length {
+        // if mean length drops below target, reset exceeds_target_at
+        if old_mean_length > self.target_mean_length && new_mean_length < self.target_mean_length {
```

Comment:
```suggestion
        if old_mean_length > self.target_mean_length && new_mean_length <= self.target_mean_length {
```

Reply:
When `new_mean_length <= self.target_mean_length` (including the case when the two are equal) , the alpha decreasing should be considered stopped I think.